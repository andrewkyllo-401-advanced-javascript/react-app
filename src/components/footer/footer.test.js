import React from 'react'
import Footer from './footer.js'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

describe('<Footer />', () => {
  it('It renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('Is alive at application start', () => {
    const component = shallow(<Footer />)
    expect(component.find('footer')).toExist()
    expect(component.find('footer')).toIncludeText('\u00A9 2018 Code Fellows')
  })
})
