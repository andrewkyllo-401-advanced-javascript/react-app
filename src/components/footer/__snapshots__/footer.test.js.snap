// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> It renders correctly 1`] = `
<footer>
  Â© 2018 Code Fellows
</footer>
`;
